packages/ui/src/common/username-field.tsx

```diff
@@ -15,10 +19,7 @@ export function UsernameField({
       <Field.Root>
         <Common.Label asChild>
           <Field.Label>
-            <span className='flex w-full items-baseline justify-between'>
-              <span>{label}</span>
-              {!props?.required && <span className='text-gray-10 text-sm font-medium'>Optional</span>}
```

Comment:
@alexcarpenter I've temporarily removed this but will re-add as part of SDKI-33

Don't particularly want to expand the scope of this PR (to avoid further conflicts)